kernel-module/Kbuild

```diff
@@ -15,8 +15,13 @@ ifdef NV_BUILD_DGPU
 		-DNV_BUILD_DGPU
 	KBUILD_EXTRA_SYMBOLS=/usr/src/nvidia-535.171.04/Module.symvers
 else
-	KBUILD_EXTRA_SYMBOLS=/usr/src/nvidia/nvidia-oot/Module.symvers
-	ccflags-y += -I/usr/src/nvidia/nvidia-oot/include/
+	ifdef NV_BUILD_IGX
```

Comment:
why this? Just update the path. And put Note in the readme or here about what possibly end user has to search?

Reply:
Removed this changes.